application/common/models/Reset.php

```diff
@@ -2,10 +2,13 @@
 namespace common\models;
 
 use common\helpers\Utils;
+use GuzzleHttp\Tests\Ring\Client\Server;
```

Comment:
I don't see anywhere that this `use` statement's class is actually being used. Am I just missing it?